airsonic-main/src/main/java/org/airsonic/player/security/JWTAuthenticationProvider.java

```diff
@@ -75,13 +73,22 @@ private static boolean roughlyEqual(String expectedRaw, String requestedPathRaw)
                 return false;
             }
 
-            MapDifference<String, List<String>> difference = Maps.difference(expected.getQueryParams(),
-                    requested.getQueryParams());
+            Map<String, List<String>> left = new HashMap<>(expected.getQueryParams());
+            Map<String, List<String>> right = new HashMap<>(requested.getQueryParams());
+
+            // Remove size parameters, it's security important and sonos controller change it
```

Comment:
I don't understand this comment :/

Reply:
I put that in code ? It's ok?

"Here, test if the request come from the way but include the parameters size of request, but Sonos change the size use in request depend of client can get it. The request is never the same, and we never equals, so return false."